scripts/extract-indices.js

```diff
@@ -97,19 +97,19 @@ const createIndex = result => {
   }, {});
 };
 
-const BELT_MD_DIR = path.join(__dirname, "../pages/apis/javascript/latest/belt");
+const BELT_MD_DIR = path.join(__dirname, "../pages/apis/javascript/latest/");
 const BELT_INDEX_FILE = path.join(
   __dirname,
   "../index_data/belt_api_index.json"
 );
-const beltFiles = glob.sync(`${BELT_MD_DIR}/*.md?(x)`);
+const beltFiles = glob.sync(`${BELT_MD_DIR}{belt/*.md?(x),belt.mdx}`);
```

Comment:
This change is to put the index files of `belt` and `js` in the index, so we have a title for the page title.